### PR TITLE
Hometask #2: Components

### PR DESCRIPTION
**Pull request by MarharytaKrasikava**
- The Second [hometask](https://epam-my.sharepoint.com/personal/andrey_pavlov_epam_com/_layouts/15/onedrive.aspx?id=%2Fpersonal%2Fandrey%5Fpavlov%5Fepam%5Fcom%2FDocuments%2FAngularMentoring%2F2%5FComponents%2Epdf&parent=%2Fpersonal%2Fandrey%5Fpavlov%5Fepam%5Fcom%2FDocuments%2FAngularMentoring&originalPath=aHR0cHM6Ly9lcGFtLW15LnNoYXJlcG9pbnQuY29tLzpiOi9wL2FuZHJleV9wYXZsb3YvRWJ2a09XQ3FaTGhQa0QwTTZJSThkR1lCQ0h6d0gyVy05Q2VfRllHdFJCaVNTdz9ydGltZT1SRDJDRks1NTJFZw) implemented

- Changed project structure

### CODE REVIEW

File: src/app/video-courses/courses/courses.component.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  styleUrls: ['./courses.component.css'],
+})
+export class CoursesComponent implements OnInit {
+  courses: Course[] = [];
+  addCourseIcon = faPlus;
+
+  constructor() {}
+
+  ngOnInit(): void {
+    this.courses = [
```
Comment (Yauhenia-Zhdanovich) [inline src/app/video-courses/courses/courses.component.ts]: mocks should be placed in a separate file

File: src/app/video-courses/courses/course/course.component.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<div class="course__header">
+  <h1 class="course__header__title">Video Course {{ course.id }}: {{ course.title }}</h1>
```
Comment (Yauhenia-Zhdanovich) [inline src/app/video-courses/courses/course/course.component.html]: h1 tag can be only one for the page

File: src/app/video-courses/courses/courses.component.ts (hunk 1 above)
Comment (MarharytaKrasikava) [reply src/app/video-courses/courses/courses.component.ts]: done

File: src/app/video-courses/courses/course/course.component.html (hunk 2 above)
Comment (MarharytaKrasikava) [reply src/app/video-courses/courses/course/course.component.html]: changed to h2

**Pull request merged**